charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -310,11 +310,7 @@ spec:
         - containerPort: 8090
           name: grpc
         - containerPort: 9990
-          {{- /*
-              Rename the port to `admin-http` once `/metrics` endpoint is supported
-              to include this in the prometheus scrapes.
-          */}}
-          name: admin
+          name: admin-http
```

Comment:
Think we might need to also change the liveness and readiness probes to point to `admin-http` to fix some of the tests?